website/static/js/pages/home-page.js

```diff
@@ -1,3 +1,187 @@
 var SignUp = require('../signUp.js');
-
+require('../../css/front-page.css');
```

Comment:
All imports can be made relative to `website/static`, so you can do `require('css/front-page.css')` here.